packages/kit/src/views/Developer/pages/Gallery/Components/stories/TableGallery/index.tsx

```
@@ -0,0 +1,45 @@
+        title: '基础表格',
+        description: '展示基本的表格结构和数据',
+        element: <BasicTableDemo />,
+      },
+      {
+        title: '自定义渲染',
+        description: '使用 render 函数自定义单元格内容和样式',
+        element: <CustomRenderingDemo />,
+      },
+      {
+        title: '交互功能',
+        description: '支持行点击、长按和表头排序。演示四种排序模式：正常排序、只能升序、只能降序、完全无排序',
```

ESLint

[error] Insert `⏎·········`

(prettier/prettier)